src/app/desktop/utils/index.js

```diff
@@ -694,12 +694,13 @@ export const filterForgeFilesByVersion = (files, version) => {
   });
 };
 
-export const getFirstReleaseCandidate = files => {
+export const getFirstPreferredCandidate = (files, release) => {
+  let counter = release || 1;
+
   let latestFile = null;
-  let counter = 1;
   while (counter <= 3 && !latestFile) {
     const c = counter;
```

Comment:
You can remove `const c = counter;` and directly use the `counter` variable

Reply:
I believe this was included previously to avoid breaking the eslint no-loop-func rule:
> Function declared in a loop contains unsafe references to variable(s) 'counter'. eslint([no-loop-func](https://eslint.org/docs/rules/no-loop-func))